packages/zipkin/test/trace.test.js

```diff
@@ -119,6 +119,28 @@ describe('Tracer', () => {
     });
   });
 
+  it('should record defaultTags', () => {
+    const record = sinon.spy();
+    const recorder = {record};
+    const ctxImpl = new ExplicitContext();
+    const localServiceName = 'smoothie-store';
+    const defaultTags = {myTag: 'some random stuff', oneMore: 'more random stuff'};
```

Comment:
nit can you use realistic tag names :) often people don't know what features are used for

Reply:
will do!